--- source/vtk-dicom-0.8.10/CMake/DICOMConfig.cmake.in	2019-08-14 14:11:34.000000000 +0200
+++ source/vtk-dicom-0.8.10/CMake/DICOMConfig.cmake.in	2019-08-14 14:11:27.000000000 +0200
@@ -1,3 +1,4 @@
+set(DICOM_PGK_DIR ${CONAN_VTK_DICOM_ROOT}/lib/dicom-0.8/cmake)
 #-----------------------------------------------------------------------------
 #
 # DICOMConfig.cmake - cmake configuration file for external projects.
@@ -28,7 +29,7 @@
 set(DICOM_USE_VTKZLIB "@DICOM_USE_VTKZLIB@")

 # Dependencies
-set(DICOM_VTK_DIR "@VTK_DIR@")
+set(DICOM_VTK_DIR ${CONAN_VTK_ROOT}/lib/cmake/vtk-8.2)
 if(NOT VTK_FOUND)
   find_package(VTK PATHS "${DICOM_VTK_DIR}")
 else()
